onnx/backend/test/case/node/gelu.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import math
+
+import numpy as np
+
+import onnx
+from onnx.backend.test.case.base import Base
+from onnx.backend.test.case.node import expect
+
+
+class Gelu(Base):
+    @staticmethod
+    def export() -> None:
+        node = onnx.helper.make_node("Gelu", inputs=["x"], outputs=["y"], approximate="tanh")
+
+        x = np.array([-1, 0, 1]).astype(np.float32)
+        # expected output [-0.158808, 0., 0.841192]
+        y = x * 0.5 * (1 + np.tanh((np.sqrt(2/np.pi) * (x + 0.044715 * np.power(x, 3)))))
```

Comment:
We need a reference implementation. We can just move this computation to the reference implementation.

Reply:
Eg., see the [elu implementation](https://github.com/onnx/onnx/blob/main/onnx/reference/ops/op_elu.py)

Reply:
Sorry for the delayed response, was AFK for the last 2 weeks. Added reference op implementation to onnx/reference/ops/op_gelu.py

Reply:
It is needed when the body function is missing or if the onnx implementation is really slow. Otherwise, that should be fine since this PR https://github.com/onnx/onnx/pull/5246 was merged (yesterday).